app/assets/javascripts/application.js

```diff
@@ -77,7 +77,8 @@ function onContentLoad(){
   $('[data-table=inline]').not('.dataTable').dataTable(
       {
         "sDom": "<'row'<'col-md-6'f>r>t<'row'<'col-md-6'i><'col-md-6'p>>",
-        "sPaginationType": "bootstrap"
+        "sPaginationType": "bootstrap",
+        "order": [[ 0, 'asc' ]] // Set default order for the first column
```

Comment:
Left this on this PR even though not using data-tables (as it is a good practice to have default sort)